src/data/code-samples/en/accessibility-announcement/swift-ui.md

```diff
@@ -0,0 +1,22 @@
+# Accessibility announcement - SwiftUI
+
+In SwiftUI you can track whenever the state of your app changes to post an accessibility message by using the [`UIAccessibility`](https://developer.apple.com/documentation/uikit/uiaccessibility) object. The [`post`](https://developer.apple.com/documentation/uikit/uiaccessibility/1615194-post) method can be used to post data to assistive technologies. Set the type to [`announcement`](https://developer.apple.com/documentation/uikit/uiaccessibility/notification/1620176-announcement) and supply a `string` argument to announce something.
+
```

Comment:
You usually won't announce focus in the code sample situation. Can you create a code sample for a loading state, e.g. after pressing a button to search you post an a11y announcement that results are loading